sympy/functions/combinatorial/numbers.py

```diff
@@ -1408,6 +1408,52 @@ def _eval_is_positive(self):
         if n.is_nonnegative and n.is_integer:
             return True
 
+class lobb(Function):
+    r"""
+    The Lobb number `L_{m,n}` counts the number of ways that `n + m` open parentheses
+    and `n - m` close parentheses can be arranged to form the start of a valid sequence
+    of balanced parentheses.
+
+    Lobb numbers form a natural generalization of the Catalan numbers,
+    which count the number of complete  strings of balanced parentheses of a given length.
+    Thus, the `n^{th}` Catalan number equals the Lobb number `L_{0,n}`.
+
+    The `(m, n)^{th}` Lobb number `L_{m,n}` is given by the formula:
+
+    .. math:: L_{m,n} = \frac{2m+1}{m+n+1}\binom{2n}{m+n}
```

Comment:
Do not copy text directly from [Wikipedia](https://en.wikipedia.org/wiki/Lobb_number). This is effectively plagiarism, since it is done without attribution (and even if there were attribution, the license of the text on Wikipedia is not compatible with the SymPy license).